frontend/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "headlamp",
   "version": "0.1.0",
   "private": true,
+  "main": "electron/main.js",
```

Comment:
Hmm, this is changing the `frontend/package.json` . Do we really need this here? I would expect we wouldn't need to change frontend at all.